services/ui-src/src/additional-scss/_footer.scss

```
@@ -1,93 +1,166 @@
+    @media (55em < width <= 100em) {
+      justify-content: flex-end;
+      flex-direction: row;
+    }
+  }
+  .footerCMSBrandingLeft {
+    display: flex;
+      justify-content: center;
+      align-items: center;
+      max-width: 100%;
+      padding: 0 1rem;
+      min-width: 100%;
+      margin-bottom: 1rem;
+      @media (55em < width <= 100em) {
+        margin-bottom: 0;
+        display: block;
+        width: auto;
+        max-width: auto;
+        min-width: auto;
       }
+  }
+  .footerCMSBrandingRight {
+    display: flex;
+    flex: 0 0 75%;
+    max-width: 25rem;
+  }
+  .sedsLogo {
+    max-height: 4.875rem;
+    margin: 0 auto;
+    @media (55em < width <= 100em) {
+      margin: 0;
+    }
+  }
+  .hhsLogo { 
+    display: inline-block;
+    width: 4.375rem;
+  }
+  .hhsMedicaidLogoMobile {
+    max-width: 12.5rem;
+    margin-left: 1.25rem;
+  }
+  .hhsCopyText {
+    display: inline-block;
+    text-align: left;
+    vertical-align: top;
+    font-size: 0.875rem; // 14px
+    line-height: 21px;
+    margin-top: 0;
+  }
+  .footerCMSMedicaid {
+    display: flex;
+    justify-content: flex-end;
+  }
+  .medicaidLogo {
+    display: flex;
+    flex: 0 0 75%;
+    max-width: 75%;
+    margin-top: 0.25rem;
+    @media (55em < width <= 100em) {
+      max-width: 25rem;
+    }
+  }
+  .footerBottom {
+    min-height: 3rem;
+    background-color: $brand-primary;
+    font-size: 14;
+  }
+  .footerBottomContainer {
+    max-width: 100vw;
+    @media (55em < width <= 100em) {
+      padding: 0 2rem;
+    }
+  }
+  .footerBottomFlex {
+    color: $white;
+    display: flex;
+    padding: 1rem 0;
+    justify-content: space-between;
+    align-items: normal;
+    flex-direction: column;
+    @media (55em < width <= 100em) {
+      align-items: initial;
+      flex-direction: row;
     }
   }
-
-  .tagline {
-    color: $brand-primary;
-    font-family: "Merriweather", "Georgia", "Cambria", "Times New Roman",
-      "Times", serif;
-  }
-
-  .info {
-    clear: both;
-    font-size: 0.75rem;
-    text-align: right;
-
-    .title {
-      float: right;
-      max-width: 274px;
+  .footerBottomLinkFlex {
+    display: flex;
+    flex-direction: column;
+    @media (55em < width <= 100em) {
+      flex-direction: row;
     }
-
-    .address {
-      float: right;
-      font-weight: $font-weight-light;
-      clear: both;
+    a:first-of-type > p {
+      margin-left: 0;
     }
   }
-
-  .footer-nav {
-    float: right;
-    padding-bottom: 1.2rem;
-    .usa-footer__nav {
-      font-size: 0.85rem;
-      padding-right: 0;
-      text-align: right;
-
-      li {
-        border-left: 1px solid $black;
-
-        &:first-of-type {
-          border-left: none;
-        }
-
-        a {
-          font-weight: $font-weight-normal;
-          padding: 0;
-          text-decoration: underline;
-        }
+  .link {
+    color: $white;
+    margin: 0.5rem 0;
+    @media (55em < width <= 100em) {
+      &first-of-type {
```

`:first-of-type` is a [pseudo-class](https://developer.mozilla.org/en-US/docs/Web/CSS/:first-of-type), so it needs the `:`

```suggestion
      &:first-of-type {
```

---

Fixed. Thank you!